lib/server.js

```diff
@@ -1,73 +1,110 @@
-const express = require('express');
-const fs = require('fs');
-const quote = require('regexp-quote');
-const path = require('path');
-const manifest = require('../ssr/ssr-manifest.json');
+require("dotenv").config();
+const express = require("express");
```

Comment:
I guess I'm OK with the move from single quotes to double quotes as long as it's going to be consistent from here on out.

Reply:
That was just the default prettier config.  I suggest if we want consistent rules let's get them in a .configuration file and add it to the project.